ton/wallet/address.go

```diff
@@ -84,8 +88,8 @@ func GetStateInit(pubKey ed25519.PublicKey, version VersionConfig, subWallet uin
 			MustStoreSlice(pubKey, 256).
 			MustStoreDict(nil). // empty dict of plugins
 			EndCell()
-	case V5R1:
-		config := version.(ConfigV5R1)
+	case V5Beta:
```

Comment:
The official name will be V5R1, and the previous W5 is V5 Beta. It is worth keeping V5R1 to avoid confusing new users of the library and to communicate that the previous version of V5 will be V5 Beta. A smaller number of users used the Beta contract, who will use the library in the future.